src/item_factory.cpp

```diff
@@ -176,9 +176,11 @@ void Item_factory::finalize_pre( itype &obj )
         obj.integral_volume = obj.volume;
     }
     // for ammo and comestibles stack size defaults to count of initial charges
+    // Set max stack size to 200 to prevent integer overflow
     if( obj.stackable && obj.stack_size == 0 ) {
-        obj.stack_size = obj.charges_default();
+        obj.stack_size = std::min( obj.charges_default(), 200 );
     }
+    
```

Comment:
```suggestion

```

Reply:
You don't have eof newlines?

Reply:
There are stray whitespaces on this line.

Reply:
oh whoops thought it was the eof from first glance. 😳